cli/app.go

```diff
@@ -109,6 +112,12 @@ var app = &cli.App{
 			Aliases: []string{"vvv"},
 			Usage:   "enable debug logging",
 		},
+		&cli.BoolFlag{
+			Name:    quietFlag,
```

Comment:
[discussion] What is generally expected of a `-q` flag for a CLI tool? Should it just suppress warnings? What about "info" messages (that are arguably not _necessary_ output)? As it stands this only silences the warnings from `CheckUpdateAction`, which I still think is not ideal.

Reply:
In random English words: when a CLI might be used for scripting, the quiet flag should suppress warnings not asked for by the invocation.

I don't know where exactly the CLI falls into "people script against it", but if it has any invocations that produce JSON, we should assume is going to pipe that through `jq` one day. And so the quiet flag is the lever we provide users.

Now one were using the CLI and they got a real error because of the API versioning problem and the API call simply failed -- it's fine to output that as non-json. The script is broken anyways.

Reply:
would it be appropriate then to use this PR as a first step towards having a `-q` that will quiet those? Then as we find issues that could benefit from the quiet flag we can iterate on it?

Reply:
Yep, no need to do any additional work.